src/main/java/com/amazon/opendistroforelasticsearch/sql/parser/ElasticSqlExprParser.java

```diff
@@ -622,6 +624,14 @@ public final SQLExpr primaryRest(SQLExpr expr) {
                     + lexer.pos());
         }
 
+        if (lexer.token() == Token.DOT && expr instanceof SQLIdentifierExpr) {
```

Comment:
Please add a comment here to explain the reason behind, ex. Druid parser creates SQLPropertyExpr (owner=xxx, name=xxx) later (dotRest?) for SQLIdentifier followed by DOT which causes xxx problem ...

Reply:
@dai-chen Ok will do. Thanks!